packages/usa-icon/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@uswds/usa-icon",
+  "version": "0.0.1-alpha.0",
+  "description": "USWDS icon library",
+  "main": "_index.scss",
+  "scripts": {
+    "test:js": "gulp buildSpriteStandalone && mocha './src/test/*.spec.js'",
+    "test": "npm run test:js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/uswds/uswds.git"
+  },
+  "keywords": [
+    "USWDS",
+    "USA",
+    "Icon",
+    "Icons"
+  ],
+  "author": "USWDS",
+  "license": "SEE LICENSE IN LICENSE.MD",
+  "bugs": {
+    "url": "https://github.com/uswds/uswds/issues"
+  },
+  "homepage": "https://github.com/uswds/uswds#readme",
+  "dependencies": {
+    "@uswds/uswds-core": "^0.0.1-alpha.0"
+  }
```

Comment:
Do you think we should include gulp and any script dependencies as devDeps here?

Reply:
Good catch. Added gulp devDependencies in [982043b](https://github.com/uswds/uswds/pull/4878/commits/982043b44257526c28f93978de3caaa3d68f6546)